checklist.bs

```diff
@@ -76,3 +76,72 @@ for the status code 202-case)
     implements mandatory server side <strong>pagination</strong> as specified in [Pathfinder Technical Specification V2](https://wbcsd.github.io/data-exchange-protocol/v2/#api-action-list-pagination)</br>
     <input type=checkbox> Host System has implemented <strong>Product Footprint Lifecycle events</strong> and rules as
     specified in Pathfinder Technical Specification V2 [Section 5](https://wbcsd.github.io/data-exchange-protocol/v2/#lifecycle)
+
+# Test Cases for PACT Conformance Testing # {#test-cases}
+
+## Action Authenticate (Mandatory) ## {#test-authenticate}
+
+Issue: explain that, prior to authentication, the `AuthEndpoint` has to be determined and that this
+can be done through OpenID or defaults to `auth/token`; explain also that in case a custom `AuthEndpoint` is used, the
+endpoint `auth/token` must nevertheless be exposed for backwards compatibility.
+
+### Test Case: Successful Authentication ### {#test-authenticate-success}
+
+Prior to this test, the test target host system must provide the testing party with the necessary
+credentials (i.e., a `client_id` and a `client_secret`).
+
+#### Request #### {#test-authenticate-success-request}
+
+An athentication request must be sent to the `AuthEndpoint` of the test target host system with
+**correct credentials** and the following syntax:
+
+<pre highlight=http>
+POST <a href="https://wbcsd.github.io/tr/data-exchange-protocol/#authendpoint">AuthEndpoint</a> HTTP/1.1
```

Comment:
I believe this option might be overkill and hard to maintain. Please see the comment below for what I consider a better alternative.